lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -658,14 +797,16 @@ class InputConfiguration {
   /// supported by Safari.
   final bool autocorrect;
 
+  final bool enableDeltaModel;
+
   final AutofillInfo? autofill;
 
   final EngineAutofillForm? autofillGroup;
 
   final TextCapitalizationConfig textCapitalization;
 }
 
-typedef OnChangeCallback = void Function(EditingState? editingState);
+typedef OnChangeCallback = void Function(EditingState? editingState, TextEditingDeltaState? editingDeltaState);
```

Comment:
I think it makes sense to add this parameter rather than creating a new type as long as nothing is broken by it.